VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1362,7 +1430,7 @@ public void onDismiss() {
                         hideTransparentImage();
                         if(player!=null){
                             player.getController().setSettingsBtnDrawable(false);
-                            player.getController().setAutoHide(true);
+                            player.getController().setAutoHide(!getTouchExploreEnabled());
```

Comment:
I'm not sure what happened here. I created the setAutoHide() method. At first I set it to true just to test. But either way, yes it is intended.